sdk/azidentity/interactive_browser_credential.go

```diff
@@ -144,3 +146,11 @@ func interactiveBrowserLogin(authorityHost string, tenantID string, clientID str
 		redirectURI: redirectURL,
 	}, nil
 }
+
+// if the authority host ends with a "/"" remove it and return
+func checkAuthHostFormat(s string) string {
```

Comment:
There is `azcore.JoinPaths()` which will handle URL path concatenation and trim any extra `/` as required.

Reply:
Fixed. I parsed the authority host string and then joined the paths.

Reply:
Can this func be removed now?

Reply:
Yes, it's done now.